apps/users/handlers.py

```diff
@@ -112,8 +137,31 @@ async def users_list(university_id: int, user=Depends(get_current_user)):
     }
 
 
-@users_router.post("/{university_id}/users/", response_model=JSENDOutSchema[CreateUserOut], tags=["SuperAdmin dashboard"])
+@users_router.post("/{university_id}/users/",
+                   name="create_user",
+                   response_model=JSENDOutSchema[CreateUserOut],
+                   summary="Create university user",
+                   responses={
+                       200: {"description": "Successful create university user response"},
+                       422: {"model": JSENDFailOutSchema, "description": "ValidationError"}
+                   },
+                   tags=["SuperAdmin dashboard"])
 async def create_user(university_id: int, user: CreateUserIn, auth=Depends(get_current_user)):
+    """
+        **Create university user**
+
+        **Path**:
+        - **university_id**: university id for creating user
+
+        **Input**:
+        - **email**: user email, only letters (a-z), numbers (0-9) and periods (.) are allowed, required
+        - **password**: password, cannot be empty, required
+        - **password_re_check**: password recheck, required
+        - **role_id**: user role id, 1 - student, 2 - admin, 3 - superadmin, required
```

Comment:
it's not a good idea to describe role values there is bcs it can be changed lately

Reply:
We can everything change lately

Reply:
sure, but anyway, data must store in db, not in api
please, delete list of roles